addons/l10n_ro/res.bank.csv

```diff
@@ -0,0 +1,40 @@
+"id","active","name","bic","street","city","state/id","country/id","phone"
+"base.res_bank_1","TRUE","Alpha Bank Romania","BUCUROBU","Calea Dorobantilor 237 B, sector1","BUCURESTI","RO_B","base.ro","(021) 209.99.99"
```

Comment:
the bank `base.res_bank_1` [already exists](https://github.com/odoo/odoo/blob/dbb0b42695b987894a73d0b78a8c9de43fefac28/openerp/addons/base/res/res_currency_data.xml#L456), this means you are modifying it